src/pages/rsvp/showandtell.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react'
+import { Layout, Section, SectionContent } from '../../components/layout'
+import * as S from '../../page-components/projects/styles'
+import { Input } from '../../components/inputs'
+import { Button } from '../../components/buttons'
+import { FormikErrors, useFormik } from 'formik'
+import { EMAIL_REGEX } from '../../../api/newsletter'
+import { PageProps } from 'gatsby'
+
+const submitUrl =
+  'https://cd-tools-git-feature-rsvp-ceskodigital.vercel.app/rsvp'
+
+interface ShowAndTellFormValues {
+  email: string
+}
+
+const ShowAndTell: React.FC<PageProps> = ({ location }) => {
+  const [registered, setRegistered] = useState(false)
+  const showEmail = new URLSearchParams(location.search).get('id') == null
+
+  const validate = (
+    values: ShowAndTellFormValues
+  ): FormikErrors<ShowAndTellFormValues> => {
+    if (showEmail) {
+      if (!EMAIL_REGEX.test(values.email)) {
+        return {
+          email: 'Nevalidní email',
+        }
+      }
+    }
+    return {}
+  }
+
+  const onSubmit = (values: ShowAndTellFormValues) => {
+    if (!registered) {
+      // Fire&Forget
+      setRegistered(true)
+
+      const body = showEmail
+        ? values
+        : {
+            id: new URLSearchParams(location.search).get('id'),
+          }
+
+      fetch(submitUrl, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(body),
+      })
+    }
+  }
+
+  const form = useFormik<ShowAndTellFormValues>({
+    initialValues: {
+      email: '',
+    },
+    validate,
+    onSubmit,
+  })
+
+  return (
+    <Layout
+      crumbs={[]}
+      seo={{
+        title: '',
+        description: '',
+      }}
+    >
+      <Section>
+        <SectionContent>
+          <S.Heading>Show&Tell</S.Heading>
+          <S.Tagline>Bude to super!</S.Tagline>
```

Comment:
Nevím, jak dlouho stránka bude žít. Pokud by měla být dlouhodobě, tak bych byl za přidání textů do překladů.

Reply:
Dobry postreh. Tohle je proof of concept pro Show&Tell. Do budoucna bude vice eventu (napriklad cEdu webinare a podobne eventy) definovanych v Airtable a RSVP stranky pro ne generovane s veskerym obsahem. To je verze 1.1